src/rf/create-aws-resources.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from os import environ
+
+from django.conf import settings
+import boto3
+import botocore
+
+SQS_REGION = environ.get('RF_SQS_REGION')
+SQS_QUEUE = environ.get('RF_SQS_QUEUE')
+SQS_DEAD_LETTER_QUEUE = environ.get('RF_SQS_DEAD_LETTER_QUEUE')
+S3_BUCKET = environ.get('RF_S3_BUCKET')
+
+VISIBILITY_TIMEOUT = 120
+MAX_RECEIVE_COUNT = 5
+
+def make_sqs_policy(queue_arn, bucket_name):
+    """
+    Policy that allows S3 bucket to publish to SQS queue.
+    from http://docs.aws.amazon.com/AmazonS3/latest/dev/
+    ... ways-to-add-notification-config-to-bucket.html
+    """
+
+    return {
+        "Version":"2008-10-17",
+        "Id":"example-ID",
+        "Statement": [
+            {
+                "Sid":"example-statement-ID",
+                "Effect":"Allow",
+                "Principal": {
+                    "AWS":"*"
+                },
+                "Action": [
+                    "SQS:SendMessage"
+                ],
+                "Resource": queue_arn,
+                "Condition": {
+                    "ArnLike": {
+                        "aws:SourceArn":"arn:aws:s3:*:*:" + bucket_name
+                    }
+                }
+            }
+        ]
+    }
+
+class CreateAwsResources(object):
+    """"
+    Encapsulates a process for creating an SQS queue,
+    an attached dead letter queue, and an S3 bucket
+    that sends messages to the SQS queue on object creation.
+    """
+    def __init__(self):
+        print('create-aws-resources.py started')
+        import sys
+        print('create-aws-resources.py started', file=sys.stderr)
```

Comment:
That can be deleted. I put it there just in case I wasn't capturing from stdout.